python/src/dataset.rs

```diff
@@ -165,18 +166,16 @@ impl Dataset {
         let mut params = ReadParams {
             index_cache_size: index_cache_size.unwrap_or(DEFAULT_INDEX_CACHE_SIZE),
             metadata_cache_size: metadata_cache_size.unwrap_or(DEFAULT_METADATA_CACHE_SIZE),
-            session: None,
```

Comment:
Session is initialized in `Default::default()`?

Reply:
Yes